com.unity.multiplayer.mlapi/Tests/Runtime/BaseMultiInstanceTest.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Object = UnityEngine.Object;
+
+namespace MLAPI.RuntimeTests
+{
+    public abstract class BaseMultiInstanceTest
+    {
+        protected GameObject m_PlayerPrefab;
+        protected NetworkManager m_ServerNetworkManager;
+        protected NetworkManager[] m_ClientNetworkManagers;
+
+        protected abstract int NbClients { get; }
+
+        [UnitySetUp]
+        public virtual IEnumerator Setup()
+        {
+            yield return StartSomeClientsAndServerWithPlayers(true, NbClients, _ => { });
+        }
+
+        [UnityTearDown]
+        public virtual IEnumerator Teardown()
+        {
+            // Shutdown and clean up both of our NetworkManager instances
+            try
+            {
+                MultiInstanceHelpers.Destroy();
+            }
+            catch (Exception e) { throw e; }
```

Comment:
huh? do we expect exceptions to come out of `MultiInstanceHelpers.Destroy();` call? If so, we'll rethrow that exception here anyway — I'm not sure if I follow the pattern here.

Reply:
We don't want tests to affect each other. It's the finally part that's important here, so we're sure we also destroy the player prefab if there's an exception in MultiInstanceHelpers' Destroy

Reply:
gotcha!